spec/loadable_spec.rb

```diff
@@ -41,33 +41,11 @@
       end
     end
 
-    context 'With default validations disabled' do
-      before do
-        SitePrism.default_load_validations = false
-      end
-
-      after do
-        # Reset the Default Constructor so its behaviour can be modified
-        SitePrism::Page.remove_instance_variable(:@_load_validations)
-        SitePrism.default_load_validations = true
-      end
-
-      context 'A standard Page' do
-        it 'has no default load validations' do
-          klass = Class.new(SitePrism::Page)
-
-          expect(klass.load_validations.length).to eq(0)
-        end
-      end
-    end
+    context 'A standard Page' do
+      it 'has no default load validations' do
+        klass = Class.new(SitePrism::Page)
```

Comment:
hah! You said you prefer other way of doing this some time [ago](https://github.com/natritmeyer/site_prism/pull/250#discussion_r186646007) 😆 
North remembers!